src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/domain/BlogPost.java

```diff
@@ -70,6 +70,10 @@ public boolean isRejected() {
         return Boolean.FALSE.equals(approved);
     }
 
+    public boolean isModerated() {
+        return approved != null;
+    }
+
     public boolean isNotModerated() {
```

Comment:
I think after introducing `isModerated()` we don't need ~~`isNotModerated()`~~ method anymore as its occurences  can be replaced by  `!isModerated()`. Refactoring will be easy because it's used only in [`BlogPostRepositorySpec`](https://github.com/tdziurko/jvm-bloggers/pull/156/files#diff-97cee16ec747fb35c69c3be009e7366dR44).